test/cli/build.js

```diff
@@ -167,11 +167,11 @@ test('`fusion build` app with dynamic imports chunk hashing', async t => {
     'vendor file hash should not change'
   );
   // TODO(#393) Add this back https://github.com/fusionjs/fusion-cli/pull/385
```

Comment:
Nit: remove comment.

Reply:
For some reason I was thinking the bot would make me keep this comment, but that isn't the case. Will remove.